.idea/codeStyles/Project.xml

```diff
@@ -47,31 +47,9 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
-      <option name="GENERATE_INSTANCE_VARIABLES_FOR_PROPERTIES" value="ASK" />
```

Comment:
revert changes to this file please

Reply:
I can't understand how these files changed. I will revert these. Sorry for the trouble